### PR TITLE
Changelog for Clippy 1.81 :beginner:

### PR DESCRIPTION
**Pull request by xFrednet**
Roses are red,
Violets are blue,
Expectations are stable,
And reasons are set

---

### The cat of this release is *Keepy* submitted by @blyxyas:

The cats of this Clippy release

Cats for the next release can be nominated in the comments :D

---

changelog: none

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,7 +6,61 @@ document.
...
+* Added [`needless_character_iteration`] to `suspicious`
+  [#12815](https://github.com/rust-lang/rust-clippy/pull/12815)
+
+### Moves and Deprecations
+
+* [`clippy::allow_attributes`], [`clippy::allow_attributes_without_reason`]: Now work on stable
+  [rust#120924](https://github.com/rust-lang/rust/pull/120924)
+* Renamed `overflow_check_conditional` to [`panicking_overflow_checks`]
+  [#12944](https://github.com/rust-lang/rust-clippy/pull/12944)
+* Moved [`panicking_overflow_checks`] to `correctness` (From `complexity` now warn-by-default)
```
Comment (y21) [inline CHANGELOG.md:44]:
```suggestion
* Moved [`panicking_overflow_checks`] to `correctness` (From `complexity` now deny-by-default)
```

**Review by flip1995 — APPROVED** [review]
Thanks! :heart_eyes_cat:

**Comment by and-reas-se** [comment]
Hello, I would like to nominate Vera for a future release.

vera-2024

**Pull request merged**